### ISSUE TITLE
Sample log files / Tensorboard summaries

### ISSUE THREAD

**Issue by mts42000** [facebookresearch/mae#39]
It would be great to publish the log files generated by pre-training/fine-tuning/linear probe runs or their tensorboard summary files (or screenshots) for reference. 

More concretely, I am trying to replicate the base config pre-training and linear probe as closely as possible in Jax but I am getting 60% accuracy which is much lower than the expected 68%. Inspecting these logs/TB summaries will help me find the possible discrepancies.

This is the loss curve on pixel prediction (not normalized)
![image](https://user-images.githubusercontent.com/4112440/153532160-412baf70-8743-441a-a11f-08020527726f.png)

And this is the linear probe run results
![image](https://user-images.githubusercontent.com/4112440/153532952-eff3e70e-c8b5-4ede-8b34-505e9530ba7b.png)

**Comment by mts42000** [comment]
Interestingly, the exact same setup I used above but with the per patch normalized loss yields only 53.9% accuracy for linear probe.

This is the normalized pixel prediction loss curve
![image](https://user-images.githubusercontent.com/4112440/153696310-467e28c5-9380-47ee-aa5e-3a322214d86f.png)

And this is the linear probe run (53.9% accuracy)
![image](https://user-images.githubusercontent.com/4112440/153696349-ceade316-c868-4a3a-bd39-6312b5080796.png)
